frontend/src/App.js

```diff
@@ -1,16 +1,16 @@
-import "./App.css";
-import ScrollToTop from "./utils/ScrollToTop";
-import { BrowserRouter, Routes, Route } from "react-router-dom";
+import React from 'react';
```

Comment:
We already have React imported globally. importing React in other files is redundant. I'd recommend we remove this